### PR TITLE
Newsletter 7: Quest, pGLOWrpg, RoboI, RecallS, China

### PR DESCRIPTION
**Pull request by ozkriff**
_Part of #64 ("Newsletter 7: Coordination/Tracking #64")_

@seratonik @tom-leys @PsichiX, r?

### CODE REVIEW

File: content/posts/newsletter-007/index.md [hunk 1]
```diff
@@ -527,6 +527,98 @@ February's full weekly devlogs: "This Week In Veloren...":
...
+Check out stream highlights on [Tom's YouTube channel][recall-youtube].
+
+_Discussions:
+[/r/rust](https://reddit.com/r/rust/comments/f9m00v/space_factory_game_in_rust_godot_the_recall)_
+
+[Tom Leys]: https://twitter.com/RecallSingular1
+[recall-s-feb]: https://medium.com/@recallsingularity/recalling-nov-2019-236cdf9c0a8a
+[recall-youtube]: https://youtube.com/channel/UCzgUlowiaKXJiNIAi0c9Qsg/videos
+
+**TODO**: other tweets!
```
Comment (Uriopass) [inline content/posts/newsletter-007/index.md]: Looks like you forgot this todo before pushing ?

**Comment by PsichiX** [comment]
give me 2-3 hours until i get pc into my hands and i'll add oxygengine announcement :)

**Comment by ozkriff** [comment]
@PsichiX 
image
@AngelOnFira is already working on Oxygengine section (see the coordination issue).

File: content/posts/newsletter-007/index.md (hunk 1 above)
Comment (ozkriff) [reply content/posts/newsletter-007/index.md]: oops

**Comment by ozkriff** [comment]
^ @AlexEne @Lokathor @AngelOnFira what do you think?

**Comment by 17cupsofcoffee** [comment]
FWIW, if nobody else shares those concerns, I won't block this merging!

**Comment by PsichiX** [comment]
@ozkriff the reason i wanted to add something to the Oxygengine part is that i wanted to add important node about the Visual Novel and Animation modules (this note is published in rust gamedev subreddit with short movie clip), i could add that to @AngelOnFira MR, but if you think that it is not possible, i'm not gonna block this PR :)

**Comment by ozkriff** [comment]
Discussion about Oxygengine updates is moved to #78.

:timer_clock: Soo, if there're no objections, I'm merging this in a few hours, ok?

**Pull request merged**